src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509Encoder.cs

```diff
@@ -147,7 +147,19 @@ public X509ContentType GetCertContentType(string fileName)
 
         public byte[] EncodeX509KeyUsageExtension(X509KeyUsageFlags keyUsages)
         {
-            throw new NotImplementedException();
+            // The numeric values of X509KeyUsageFlags mean that if we interpret it as a little-endian
+            // ushort it will line up with the flags in the spec.
+            ushort ushortValue = unchecked((ushort)(int)keyUsages);
+            byte[] data = BitConverter.GetBytes(ushortValue);
```

Comment:
(Actually, looks like I was wrong about BitConverter's usage of IsLittleEndian... it uses it in ToUInt16 but not in GetBytes(UInt16).)